Jenkinsfile

```diff
@@ -1,10 +1,24 @@
 //https://github.com/feedhenry/fh-pipeline-library
+import org.feedhenry.Utils
 
 fhBuildNode {
+
+    def utils = new Utils()
+
     stage('Install Dependencies') {
         npmInstall {}
     }
 
+    stage('Test') {
+        gruntCmd {
+            cmd = 'format'
+        }
+        if(utils.gitRepoIsDirty()) {
```

Comment:
Unrelated: consider putting a space between `if` and the opening paren of the condition. It's not a big deal and doesn't change anything functionally, I just like it visually as a way to distinguish between function calls (no space) and language keywords (space). Feel free to ignore this though :)